chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -146,6 +146,29 @@ if test "x$MD5_ADMIN" != "x$MD5_LOCAL"; then
     exit 1
 fi
 
+# Setup groups and users
+# # ---------------
+
+if [[ -z `grep '^qemu' /etc/group` ]]; then
+  groupadd --system --gid 484 qemu
+elif [[ -z `grep '^qemu:*.:484:*.' /etc/group` ]]; then
+  echo "Group qemu did not have gid 484."
+  exit 1
+fi
+
+if [[ -z `grep '^kvm' /etc/group` ]]; then
+  groupadd --system --gid 485 kvm
+elif [[ -z `grep '^kvm:*.:485:*.' /etc/group` ]]; then
+  echo "Group kvm did not have gid 485."
+  exit 1
+fi
+
```

Comment:
Actually, repeating the same code three times, screems for moving it to a function :)